update README to mention numbering in creation

eteq opened the pull request:
In #36 we clarified that APE #s are assigned at-merge.  But it looks like we forgot to update that in an earlier part of the README.  This PR fixes that, along with a bit more clarification in that part of the README.

kelle commented on README.rst:
```diff
@@ -69,28 +69,29 @@ Accepted APEs
-PR with that file once you've written it up.  Be sure to look through the PRs in
-this repo first so that you choose an APE# that reflects both those that are in
-the repository *and* those that are under discussion.
+Just fork the repository, copy ``APEtemplate.rst`` to
```
- Just remove just. :)

kelle commented on README.rst:
```diff
@@ -69,28 +69,29 @@ Accepted APEs
-this repo first so that you choose an APE# that reflects both those that are in
-the repository *and* those that are under discussion.
+Just fork the repository, copy ``APEtemplate.rst`` to
+``APE_<some_working_name>.rst`` and issue a Pull REquest with that file once
```
REquest -> pull request

eteq commented:
Thanks - I think I've addressed @kelle's suggestions (and rebased against master). If you think this is now good, @kelle, feel free to approve and merge!
